### PR TITLE
Edit Profile by Shamjad

### PR DESCRIPTION
**Pull request by shamjadk**

### CODE REVIEW

File: lib/core/widgets/add_image_widget.dart [hunk 1]
```diff
@@ -12,11 +12,11 @@ class AddImageWidget extends ConsumerWidget {
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
-    final constants = ref.watch(profilePageContstantsProvider);
     final assets = ref.watch(appAssetsConstantsProvider);
+    final constants = ref.watch(profilePageContstantsProvider);
     final appTheme = AppTheme.of(context);
     return InkWell(
-      onTap: () => ImagePickerUtils.showDialogueForImagePicker(context),
+      onTap: () => {ImagePickerUtils.showDialogueForImagePicker(context)},
```
Comment (Sireen-Official) [inline lib/core/widgets/add_image_widget.dart]: Remove unwanted curly brackets

**Review by Sireen-Official — CHANGES_REQUESTED** [review]
Do the changes mentioned in the comments

File: lib/core/widgets/add_image_widget.dart (hunk 1 above)
Comment (shamjadk) [reply lib/core/widgets/add_image_widget.dart]: Fixed

**Review by Sireen-Official — APPROVED** [review]
Review approved

File: lib/core/widgets/add_image_widget.dart [hunk 2]
```diff
@@ -29,7 +29,7 @@ class AddImageWidget extends ConsumerWidget {
           Text(
             constants.txtAddImage,
             style: appTheme.typography.uiSemibold
-                .copyWith(color: appTheme.colors.textSubtlest),
+                .copyWith(color: appTheme.colors.textSubtlest.withOpacity(.7)),
```
Comment (thadathilsinan) [inline lib/core/widgets/add_image_widget.dart]: use text_disabled in theme

**Review by thadathilsinan — CHANGES_REQUESTED** [review]

**Review by thadathilsinan — APPROVED** [review]

**Pull request merged**